mm/context_label.py

```diff
@@ -0,0 +1,161 @@
+from typing import Tuple, Optional, Dict
+import itertools
+
+import numpy as np
+
+from i6_core.lib.hdf import get_returnn_simple_hdf_writer
+from i6_core.lib.rasr_cache import FileArchive
+from sisyphus import Job, Task, tk, gs
+from apptek_asr.artefacts.factory import AbstractArtefactRepository
+
+
+class GetContextLabelFromDenseTyingJob(Job):
```

Comment:
Since you are also reading the center phoneme information, I would not call the job `GetContextLabel`, maybe `GetPhonemeLabelsFromNoTyingDense`? Also I think `DenseTying` is not clear, since it hints to some kind of tying, whereas here is about no tying at all but complete enumeration.